envisage/ui/single_project/action/save_as_project_action.py

```diff
@@ -74,7 +74,7 @@ def _on_project_changed(self, obj, trait_name, old, new):
         if new is not None:
             self._update_project_listeners(new, remove=False)
 
-        super(SaveAsAction, self)._on_project_changed(
```

Comment:
ohh god looks like this path is never tested/exercised because the class name is `SaveAsProjectAction` and there is no `SaveAsAction`. It's possible that this class was called `SaveAsAction` earlier

Reply:
See also #269. `single_project` should just die